pages/my-profile/my-collection/index.js

```diff
@@ -0,0 +1,10 @@
+import MyCollectionList from "@/components/MyCollectionList";
+
+export default function MyCollection() {
+  return (
+    <>
+      <h1>My collection</h1>
+      <MyCollectionList />
+    </>
```

Comment:
Replacing the fragments with a  `<main>` tag helps for html structure here as well.
```suggestion
    <main>
      <h1>My collection</h1>
      <MyCollectionList />
    </main>
```